docs/DeveloperGuide.md

```diff
@@ -138,43 +139,90 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Implemented\] Schedule an assignment
+### Schedule assignments feature
 
-The user can input a deadline and expected time for an assignment to get a suggested start time and end time to work on the assignment.
-The suggested time will be within working hours from 6am to 11pm local time.
-The expected hours for an assignment ranges from 1 to 5 hours.
-The suggested time will not class with any of the suggested time for other assignments and lessons.
+The user can get a suggested schedule to work on an assignment by providing the following fields:
+- Index of the assignment in the displayed assignment list
+- Expected hours to complete the assignment
+- Date and time after which you want to start working on the assignment  
+- Date and time before which you want to finish the assignment
 
-#### Reasons for Implementation
+A suggested time slot will be provided to the users. The suggested schedule will have the following conditions:
+- Expected hours for an assignment ranges from 1 to 5 hours.
+- Suggested time slot will be within working hours from 6am to 12pm local time.
+- Suggested time slot will not clash with any of the suggested time for other assignments and lessons.
+
+It implements the following operation:
+* `schedule 1 hrs/5 af/01-01-2020 0000 by/23-12-2020 2359` - Suggest a schedule for the 1st assignment in the displayed assignment list,
+given that the time to work on the assignment is 5 hours, and the user want to do the assignment between 01-01-2020 0000 and 23-12-2020 2359.
```

Comment:
Maybe change the phrasing to "Suggests a schedule for the 1st assignment in the displayed assignment, given that the user expects to spend 5 hours to work on the assignment and wishes to complete it sometime between 01-01-2020 0000 and 23-12-2020 2359"